{"hierarchy":{"paths":[["doc:\/\/RxFireAuth\/documentation\/RxFireAuth","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError"]]},"abstract":[{"type":"text","text":"The requested action cannot be performed because there is already an anonymous user logged-in."}],"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"alreadyAnonymous"}],"platforms":["macOS"],"languages":["swift"]}],"kind":"declarations"}],"schemaVersion":{"patch":0,"major":0,"minor":3},"kind":"symbol","metadata":{"role":"symbol","title":"UserError.alreadyAnonymous","modules":[{"name":"RxFireAuth"}],"symbolKind":"case","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"alreadyAnonymous"}],"roleHeading":"Case","externalID":"s:10RxFireAuth9UserErrorO16alreadyAnonymousyA2CmF"},"identifier":{"url":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/alreadyAnonymous","interfaceLanguage":"swift"},"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/rxfireauth\/usererror\/alreadyanonymous"]}],"references":{"doc://RxFireAuth/documentation/RxFireAuth/UserError/alreadyAnonymous":{"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/alreadyAnonymous","kind":"symbol","title":"UserError.alreadyAnonymous","role":"symbol","type":"topic","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"alreadyAnonymous"}],"abstract":[{"text":"The requested action cannot be performed because there is already an anonymous user logged-in.","type":"text"}],"url":"\/documentation\/rxfireauth\/usererror\/alreadyanonymous"},"doc://RxFireAuth/documentation/RxFireAuth/UserError":{"url":"\/documentation\/rxfireauth\/usererror","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError","abstract":[{"text":"Errors thrown by  ","type":"text"},{"code":"UserManagerType","type":"codeVoice"},{"type":"text","text":" implementations."}],"title":"UserError","type":"topic","role":"symbol","kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"UserError"}],"navigatorTitle":[{"kind":"identifier","text":"UserError"}]},"doc://RxFireAuth/documentation/RxFireAuth":{"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth","kind":"symbol","role":"collection","abstract":[],"url":"\/documentation\/rxfireauth","title":"RxFireAuth","type":"topic"}}}